app/download.html

```diff
@@ -12,16 +12,17 @@ <h1 class="hero-title">$( index.download )$</h1>
         <!-- Version selector -->
         <div class="dropdown">
           <button class="btn btn-link btn-lg bg-lighter-dark text-white dropdown-toggle" type="button" id="version-select" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            v0.21.2 <img class="chevron-down" src="/static/img/icons/chevron-down.svg" alt="Chevron Down">
+            v22.0.0 <img class="chevron-down" src="/static/img/icons/chevron-down.svg" alt="Chevron Down">
```

Comment:
It should be v0.22.0 shouldn't it?

Reply:
No. We switched versioning schemes.  22.0.0 it is. 👍

Reply:
Understood